test/config/integration/server_xds.bootstrap.udpa.yaml

```diff
@@ -1,8 +1,5 @@
 dynamic_resources:
-  lds_resources_locator:
-    scheme: FILE
-    id: {{ lds_json_path }}
-    resource_type: envoy.config.listener.v3.ListenerCollection
+  lds_resources_locator: file:///{{ lds_json_path }}
```

Comment:
Note that we need to distinguish flat string that start with `file:`, `http:` as well as `xdstp:` going forward.